collector/pom.xml

```diff
@@ -42,8 +42,8 @@
         <artifactId>maven-compiler-plugin</artifactId>
         <version>2.3.2</version>
         <configuration>
-          <source>1.5</source>
-          <target>1.5</target>
+          <source>1.8</source>
```

Comment:
2/3 of our users are still on 1.6, 1.8 is barely in use. This needs to stay as-is for many years to come.